.rubocop.yml

```diff
@@ -4,9 +4,7 @@
 AllCops:
   TargetRubyVersion: 2.4
   DisplayCopNames: true
-  Include:
```

Comment:
Why it was removed?

Reply:
Because with this setting just 2 files are checked by Rubocop, is it expected behaviour?

Reply:
Then make sure old 2 files are checked by Rubocop